spec/system/llms/ai_llm_spec.rb

```diff
@@ -76,14 +76,14 @@
     it "shows the provider as CDCK in the UI" do
       visit "/admin/plugins/discourse-ai/ai-llms"
       expect(page).to have_css(
-        "[data-llm-id='cdck-hosted'] .column-provider",
```

Comment:
Noticed that we are treating the table wrapper classes as extraneous (e.g. `admin-row__detail`, `d-admin-row__controls`) during the assertions. Is this to ensure the tests aren't too fragile? Assertions on those would enforce that these elements keep to classes that align with the admin UI guidelines.